README.md

```diff
@@ -61,4 +61,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](https://github.com/CRClothier/My-Portfolio-Website/blob/main/LICENSE) licensed.
```

Comment:
- [ ] **[OPTIONAL]** I can notice your efforts in trying to add the MIT license. But I'd like you to do your own MIT license instead of using somebody else's. I'm approving it only because your project met the other criteria, and due to the fact that it's Saturday night and you might not get another reviewer.

Please avoid doing this again to avoid getting your project rejected. Remember: if the person that you copied the MIT from decides to delete their repo, you won't have your MIT anymore.

Other than that, your project is looking great :sunglasses: